### PR TITLE
feat: document nRF Cloud Cell Geolocation Service

### PR DESCRIPTION
**Pull request by coderbyheart**
See https://github.com/NordicSemiconductor/asset-tracker-cloud-docs/discussions/9

### CODE REVIEW

File: docs/azure/UnwiredLabsCellGeolocation.rst [hunk 1]
```diff
@@ -5,25 +5,32 @@ Unwired Labs Cell Geolocation
...
+            appName=${APP_NAME:-nrfassettracker} \
+            appRegistrationClientId=$APP_REG_CLIENT_ID \
+            b2cTenant=${B2C_TENANT:-nrfassettrackerusers} \
+            keyVaultName=${APP_NAME:-nrfassettracker} \
+            enableUnwiredLabsCellLocation=true
 
 This command enables the `geolocateCellFromUnwiredLabs` function to resolve cells.
 Otherwise, this function returns a ``402`` status on the API route ``cellgeolocation/unwired``.
 
+The API key needs to be stored in the key vault:
```
Comment (peknis) [inline docs/azure/UnwiredLabsCellGeolocation.rst]:
```suggestion
Store the API key into the key vault as follows:
```

File: docs/azure/nRFCloudLocationServices.rst [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+            appRegistrationClientId=$APP_REG_CLIENT_ID \\
+            b2cTenant=${B2C_TENANT:-nrfassettrackerusers} \\
+            keyVaultName=${APP_NAME:-nrfassettracker} \\
+            enableNrfCloudCellLocation=true \\
+            nrfCloudTeamId=*your team ID*
+
+This command enables the `geolocateCellFromNrfCloud` function to resolve cells.
+Otherwise, this function returns a ``402`` status on the API route ``cellgeolocation/nrfcloud``.
+
+The service key needs to be stored in the key vault:
```
Comment (peknis) [inline docs/azure/nRFCloudLocationServices.rst]:
```suggestion
Store the service key into the key vault as follows:
```

File: docs/azure/UnwiredLabsCellGeolocation.rst [hunk 3]
```diff
@@ -5,25 +5,32 @@ Unwired Labs Cell Geolocation
...
-            b2cTenant=${B2C_TENANT:-nrfassettrackerusers} \\
-            unwiredlabsApiKey=*your API key*
+        --parameters \
+            appName=${APP_NAME:-nrfassettracker} \
+            appRegistrationClientId=$APP_REG_CLIENT_ID \
+            b2cTenant=${B2C_TENANT:-nrfassettrackerusers} \
+            keyVaultName=${APP_NAME:-nrfassettracker} \
+            enableUnwiredLabsCellLocation=true
 
 This command enables the `geolocateCellFromUnwiredLabs` function to resolve cells.
```
Comment (peknis) [inline docs/azure/UnwiredLabsCellGeolocation.rst]: Should we have the function name between double back ticks to display it as `geolocateCellFromUnwiredLabs`?

File: docs/azure/nRFCloudLocationServices.rst [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+        --template-file azuredeploy.json \\
+        --parameters \\
+            appName=${APP_NAME:-nrfassettracker} \\
+            appRegistrationClientId=$APP_REG_CLIENT_ID \\
+            b2cTenant=${B2C_TENANT:-nrfassettrackerusers} \\
+            keyVaultName=${APP_NAME:-nrfassettracker} \\
+            enableNrfCloudCellLocation=true \\
+            nrfCloudTeamId=*your team ID*
+
+This command enables the `geolocateCellFromNrfCloud` function to resolve cells.
```
Comment (peknis) [inline docs/azure/nRFCloudLocationServices.rst]: Same question as above. How do we want to display the function name?

File: docs/azure/nRFCloudLocationServices.rst [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+nRF Cloud Location Services
+###########################
+
+Cell Location Service
+*********************
+
+You can enable the :ref:`cell geolocation lookup <app-cellgeolocation>` for single cells and neighboring cell reports for the application using `nRF Cloud's Cell Location Service API <https://api.nrfcloud.com/v1#tag/Cell-Position>`_ in your deployment.
+
+To use nRF Cloud's Cell Location Service API, set the ``enableNrfCloudCellLocation`` parameter to ``true`` and set the ``nrfCloudTeamId`` parameter to your team ID when deploying the solution.
```
Comment (peknis) [inline docs/azure/nRFCloudLocationServices.rst]:
```suggestion
To use API, set the ``enableNrfCloudCellLocation`` parameter to ``true`` and set the ``nrfCloudTeamId`` parameter to your team ID when deploying the solution.
```
I believe you don't need to repeat the full name of the API as you just mentioned that in the previous chapter.

**Review by peknis — CHANGES_REQUESTED** [review]

File: docs/azure/UnwiredLabsCellGeolocation.rst (hunk 3 above)
Comment (coderbyheart) [reply docs/azure/UnwiredLabsCellGeolocation.rst]: Right, this is markdown, not restructuredtext.
```suggestion
This command enables the ``geolocateCellFromUnwiredLabs`` function to resolve cells.
```

File: docs/azure/nRFCloudLocationServices.rst (hunk 4 above)
Comment (coderbyheart) [reply docs/azure/nRFCloudLocationServices.rst]:
```suggestion
This command enables the ``geolocateCellFromNrfCloud`` function to resolve cells.
```

File: docs/azure/nRFCloudLocationServices.rst (hunk 5 above)
Comment (coderbyheart) [reply docs/azure/nRFCloudLocationServices.rst]:
```suggestion
To use the API, set the ``enableNrfCloudCellLocation`` parameter to ``true`` and set the ``nrfCloudTeamId`` parameter to your team ID when deploying the solution.
```

**Review by peknis — APPROVED** [review]

**Comment by coderbyheart** [comment]
Sorry @peknis for re-request.

**Comment by peknis** [comment]
> Sorry @peknis for re-request.

No worries. We were both quicker than a greased lightning.

**Review by peknis — APPROVED** [review]

File: docs/app/CellGeolocation.rst [hunk 6]
```diff
@@ -51,6 +51,7 @@ They provide the geolocation of nearly every cell tower and with this informatio
 Follow the configuration guide in the respective implementation to enable nRF Cloud's cell geolocation API:
 
 * :ref:`AWS <aws-nrf-cloud-location-services>`
+* :ref:`AWS <azure-nrf-cloud-location-services>`
```
Comment (umapraseeda) [inline docs/app/CellGeolocation.rst]: Shouldn't it be Azure?
```suggestion
* :ref:`Azure <azure-nrf-cloud-location-services>`
```

Comment (coderbyheart) [reply docs/app/CellGeolocation.rst]: Yes, good catch. Copy and Paste mistake.

Comment (peknis) [reply docs/app/CellGeolocation.rst]: This is why two pairs of eyes are needed for a review. I totally missed this.

**Review by umapraseeda — APPROVED** [review]

**Pull request merged**